tests/test_broken_functions.py

```diff
@@ -1,5 +1,6 @@
 from azure.functions_worker import protos
 from azure.functions_worker import testutils
+import traceback
```

Comment:
nit: Move to the top of the file as standard library imports come before custom imports by convention (https://www.python.org/dev/peps/pep-0008/#imports)